index.html

```diff
@@ -2998,20 +2997,137 @@ <h4>tactileGraphics</h4>
 				</section>
 			</section>
 		</section>
+		<section id="app-media-types" class="appendix">
+			<h2>Media type registrations</h2>
+
+			<section id="app-media-type">
+				<h3>The <code>application/ebrl+zip</code> media type</h3>
+
+				<p>This appendix registers the media type <code>application/ebrl+zip</code> for the eBraille
+					implementation of the EPUB Open Container Format (OCF).</p>
+
+				<p>The OCF ZIP container file is based on the zip archive format (see <a
+						href="https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT"
+						>https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT</a>). It is used to encapsulate an
+					eBraille publication.</p>
+
+				<dl class="variablelist">
+					<dt>MIME media type name:</dt>
+					<dd>
+						<p>
+							<code>application</code>
+						</p>
+					</dd>
+
+					<dt>MIME subtype name:</dt>
+					<dd>
+						<p>
+							<code>ebrl+zip</code>
+						</p>
+					</dd>
+
+					<dt>Required parameters:</dt>
+					<dd>
+						<p>None.</p>
+					</dd>
+
+					<dt>Optional parameters:</dt>
+					<dd>
+						<p>None.</p>
+					</dd>
+
+					<dt>Encoding considerations:</dt>
+					<dd>
+						<p>OCF ZIP container files are binary files encoded in the <a class="media-type"
+								href="https://www.iana.org/assignments/media-types/application/zip">
+								<code>application/zip</code></a> media type.</p>
+					</dd>
+
+					<dt>Security considerations:</dt>
+					<dd>
+						<p>All processors that read OCF ZIP container files should rigorously check the size and
+							validity of data retrieved.</p>
+						<p>In addition, because of the various content types that can be embedded in OCF ZIP container
+							files, <code>application/epub+zip</code> may describe content that poses security
```

Comment:
this should be ebrl+zip right?

Reply:
Oops, good catch! Missed that one in the copy of the text.